packages/demo/backend/src/controllers/wallet.ts

```diff
@@ -118,4 +120,24 @@ export class WalletController {
       )
     }
   }
+
+  async getBalance(c: Context) {
+    try {
+      const validation = validateUserParams(c)
```

Comment:
🤔 In another PR, we should make this endpoint specific and error when params are missing.